blog_system/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib import admin
+from django.urls import path
+from django.urls.conf import include
+from debug_toolbar.toolbar import debug_toolbar_urls
+
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('blog/', include('blog.urls')),
+    path("accounts/", include("django.contrib.auth.urls")),
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) + debug_toolbar_urls()
```

Comment:
The `debug_toolbar_urls()` function is not typically called directly in the `urlpatterns` list. Instead, it's common to include it using Django's `include()` function, like this: `path('__debug__/', include(debug_toolbar.urls))`. Also, consider including these URLs only when `DEBUG` setting is `True`.